### PR TITLE
feat(agent): remove opcode handler for cufa when using OAPI

### PR DESCRIPTION
**Pull request by ZNeumann**

### CODE REVIEW

**Review by zsistla — APPROVED** [review]
Thanks!

**Review by mfulb — APPROVED** [review]

File: agent/php_execute.c [hunk 1]
```diff
@@ -29,9 +29,9 @@
 #include "util_url.h"
 #include "util_metrics.h"
 #include "util_number_converter.h"
-#include "php_execute.h"
```
Comment (bduranleau-nr) [inline agent/php_execute.c:32]: Good eye!

File: agent/php_execute.h [hunk 2]
```diff
@@ -64,6 +64,8 @@ extern void nr_framework_create_metric(TSRMLS_D);
  *           This is necessary to correctly instrument frameworks and libraries
  *           that are preloaded.
  */
+
 extern void nr_php_user_instrumentation_from_opcache(TSRMLS_D);
 
+#include "php_observer.h"
```
Comment (bduranleau-nr) [inline agent/php_execute.h:70]: I would suggest placing this at the top of the file after `#define PHP_EXECUTE_HDR` so it's more visible at a glance.

File: agent/php_includes.h [hunk 3]
```diff
@@ -53,6 +53,10 @@
 #define ZEND_8_0_X_API_NO 20200930
 #define ZEND_8_1_X_API_NO 20210902
 
+#if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP8+ */
```
Comment (bduranleau-nr) [inline agent/php_includes.h:56]: Maybe add `OVERWRITE_ZEND_EXECUTE_DATA` so we know this functionality is unique to OAPI?

**Review by bduranleau-nr — APPROVED** [review]
👍

**Comment by ZNeumann** [comment]
Closing because this initial implementation overlooked too many necessary changes for discussion on this PR to be relevant.

**Pull request closed without merging**